app/assets/javascripts/components/article_finder/article_finder.jsx

```diff
@@ -216,14 +216,20 @@ const ArticleFinder = createReactClass({
       delete keys.grade;
     }
 
-    if (!this.props.course_id) {
+    if (!this.props.course_id || !this.props.current_user.id) {
       delete keys.tools;
     }
 
     let list;
     if (this.state.isSubmitted && !this.props.loading) {
       const elements = _.map(this.props.articles, (article, title) => {
-        const assignment = _.find(this.props.assignments, { article_title: title });
+        let assignment;
+        if (this.props.current_user.isInstructor) {
```

Comment:
I think what we want here is to use `isNonstudent` (for instructors _and_ admins) vs `isStudent`.